null_safety/calculate_lix/bin/main.dart

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2020, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:calculate_lix/lix.dart';
+
+void main(List<String> arguments) {
+  // Parse the arguments; we expect a single argument containing the file name.
+  if (arguments.length != 1) {
+    printUsage();
+    exit(64);
+  } else {
+    // Get the file name.
+    //
+    // TIP: We're using null safe versions of the core libraries, and we already
```

Comment:
This TIP feels a little misleading. I'd word it as:

// TIP: We're using null safe versions of the core libraries, and the type
// of arguments is List<String> which means that arguments cannot contain null elements.
// As a result, the Dart analyzer knows that fileName isn't null